lynx/common/square.py

```diff
@@ -0,0 +1,28 @@
+
+from dataclasses import dataclass, field
+from typing import List, NoReturn
+
+from lynx.common.object import Object
+
+# Class wraping logic of ground and objects placed on it
+@dataclass
+class Square:
```

Comment:
How about renaming `Square` to `Field`?

Reply:
Thought about that but it's more correct in English to use square rather than field I think. On the chess board nobody uses word field but everyone says square.

Reply:
Yep, after brief thinking i think it makes sense